src/dags/bouwstroompunten.py

```diff
@@ -89,89 +91,98 @@ def get_data():
     mkdir = BashOperator(task_id="mkdir", bash_command=f"mkdir -p {tmp_dir}")
```

Comment:
If you are using more modern operators might I suggest you you the `mk_dir` task here?

```python
from common.path import mk_dir
...
mkdir = mk_dir(tmp_dir, clean_if_exists=True)
```
Not sure if you want it cleaned though. If not, leave the argument out (default value is False), or set it explicitly to false.

The task `mk_dir` is our first usage of the Airflow 2.0 Flow based API. Much more convenient API than the Operator based one. See also the `anpr` DAG that already uses `mk_dir`

Reply:
Nice addition, didn't know that!